models/user_input.js

```diff
@@ -6,7 +6,7 @@ module.exports = (db, Validator) => {
         findAll: function () {
             return new Promise((resolve, reject) => {
                 _db.query(`SELECT *, user_inputs.* FROM user_inputs
-                INNER JOIN sanitized_users on (user_inputs.device_id = sanitized_users.id OR user_inputs.user_id = sanitized_users.id )
+                INNER JOIN sanitized_users on (user_inputs.device_id = sanitized_users.id AND user_inputs.user_id = sanitized_users.id )
```

Comment:
Does this work properly?

Reply:
Yeah, I found a bug in it last night... I'll get to it soon